packages/core/src/Stores/client-store.js

```diff
@@ -263,6 +263,7 @@ export default class ClientStore extends BaseStore {
             is_financial_assessment_needed: computed,
             is_authentication_needed: computed,
             is_identity_verification_needed: computed,
+            is_poa_older_than_six_months: computed,
```

Comment:
Instead of this computed value, can we create a hook for this?

Reply:
Added it to the already created account status hook

Reply:
cant we get this information from hooks directly.
Do we need this computed value then?

Reply:
Hooks will not work since it is a JS component